tensorflow/go/operation.go

```diff
@@ -123,6 +148,67 @@ func (p Output) c() C.TF_Output {
 
 func (p Output) canBeAnInput() {}
 
+// NumConsumers returns the number of inputs that consume this output.
+func (p Output) NumConsumers() int {
```

Comment:
Do we need both a `NumConsumers` and a `Consumers`?
I realize that `NumConsumers()` here has an implementation that is more efficient than `len(Consumers())`, but I'd wager that the performance difference isn't material.

So while I'm open to opposition, right now I feel that we're better off with a smaller API surface (and doing away with `NumConsumers` completely).

Reply:
Makes sense. Removed `NumConsumers()`.